apps/flo-cli/src/cli/workflows/run.command.old.ts

```diff
@@ -0,0 +1,61 @@
+import { writeFileSync } from 'fs'
+import { indent } from '../../lib/utils'
+import { Workflow } from '../../lib/workflows/workflow.schemas'
+
+// This file is left over because we still need to add the NpmStep back into the system
```

Comment:
Lets clean it up then.

Reply:
Lets just kill the file, we can always come back to this revision if we need more reference.